module/files/helper.js

```diff
@@ -335,4 +338,177 @@ module.exports = class FilesHelper {
       }
     });
   }
+  /**
+   * Get all signed urls.
+   * @method
+   * @name preSignedUrls
+   * @param {Array} payloadData       - payload for files data.
+   * @param {String} referenceType    - reference type
+   * @param {String} userId           - Logged in user id.
+   * @param {String} templateId       - certificateTemplateId.
+   * @param {Boolean} serviceUpload     - serive Upload  {true/false}
+   * @returns {Array}                 - consists of all signed urls files.
+   */
+
+  static preSignedUrls(payloadData, referenceType, userId = '', serviceUpload = false) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let payloadIds = Object.keys(payloadData);
+
+        let result = {
+          [payloadIds[0]]: {},
+        };
+
+        let folderPath = '';
+
+        if (referenceType == messageConstants.common.DHITI) {
+          folderPath = 'reports/';
+        } else if (referenceType == messageConstants.common.CERTIFICATE) {
```

Comment:
@borkarsaish65 elseif condition actually not required. Because samiksha service does not expect certificate related reference type. it will be managed by project-service. Hence ml-core service was managing this for both survey and project

Reply:
Done